tests/msbuild/Xamarin.MacDev.Tests/DotnetTest.cs

```diff
@@ -1,3 +1,4 @@
+#if ENABLE_DOTNET
```

Comment:
Instead of conditionally compiling the file, you can do something like this in each test, which is how we usually disable tests according to build configuration:

```csharp
if (!Configuration.IncludeDotNet) {
    Assert.Ignore (".NET tests not enabled");
}
```

and also add IncludeDotNet variable to the Configuration class and load it. See `include_device` for an example: https://github.com/xamarin/xamarin-macios/blob/7c9cf9c3783d6c2f5a32cb41171e6a0f3c1f80c8/tests/common/Configuration.cs#L272

Reply:
let me be lazy! I'll do it ;)